src/Jackett/Indexers/Norbits.cs

```diff
@@ -0,0 +1,878 @@
+            if (force)
+            {
+                // Deleting Provider Storage folder and all files recursively
+                Output("\nDeleting Provider Storage folder and all files recursively ...");
+
+                // Check if directory exist
+                if (System.IO.Directory.Exists(Directory))
+                {
+                    // Delete storage directory of provider
+                    System.IO.Directory.Delete(Directory, true);
+                    Output("-> Storage folder deleted successfully.");
+                }
+                else
+                {
+                    // No directory, so nothing to do
+                    Output("-> No Storage folder found for this provider !");
+                }
+            }
+            else
+            {
+                var i = 0;
+                // Check if there is file older than ... and delete them
+                Output("\nCleaning Provider Storage folder... in progress.");
+                System.IO.Directory.GetFiles(Directory)
+                .Select(f => new System.IO.FileInfo(f))
+                .Where(f => f.LastAccessTime < DateTime.Now.AddMilliseconds(-Convert.ToInt32(ConfigData.HardDriveCacheKeepTime.Value)))
+                .ToList()
+                .ForEach(f => {
+                    Output("Deleting cached file << " + f.Name + " >> ... done.");
+                    f.Delete();
+                    i++;
+                });
+
+                // Inform on what was cleaned during process
+                if (i > 0)
+                {
+                    Output("-> Deleted " + i + " cached files during cleaning.");
+                }
+                else {
+                    Output("-> Nothing deleted during cleaning.");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Generate a random fake latency to avoid detection on tracker side
+        /// </summary>
+        private void LatencyNow()
+        {
+            // Need latency ?
+            if (Latency)
+            {
+                var random = new Random(DateTime.Now.Millisecond);
+                var waiting = random.Next(Convert.ToInt32(ConfigData.LatencyStart.Value),
+                    Convert.ToInt32(ConfigData.LatencyEnd.Value));
+                Output("\nLatency Faker => Sleeping for " + waiting + " ms...");
+
+                // Sleep now...
+                System.Threading.Thread.Sleep(waiting);
+            }
+            // Generate a random value in our range
+        }
+
+        /// <summary>
+        /// Find torrent rows in search pages
+        /// </summary>
+        /// <returns>JQuery Object</returns>
+        private CQ FindTorrentRows()
+        {
+            // Return all occurencis of torrents found
+            //return _fDom["#content > table > tr"];
+            return _fDom["# torrentTable > tbody > tr:not(:first)"];
+        }
+
+        /// <summary>
+        /// Format Date to DateTime
+        /// </summary>

```

Comment:
As this function is in french, i doubt you need her to process results of your provider ?